eth/handler.go

```diff
@@ -897,6 +897,8 @@ func (pm *ProtocolManager) txBroadcastLoop() {
 
 	for {
 		select {
+		case <-pm.quitSync:
```

Comment:
I'm not sure it's the necessary change.

In the `Stop` function, we will `Unsubscribe` the subscriptions and `pm.txsSub.Err()` will be activated.